pkg/controllers/cassandra/nodepool/util.go

```diff
@@ -0,0 +1,27 @@
+package nodepool
+
+import (
+	"github.com/coreos/go-semver/semver"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/jetstack/navigator/pkg/apis/navigator/v1alpha1"
+)
+
+func cassImageToUse(spec *v1alpha1.CassandraClusterSpec) *v1alpha1.ImageSpec {
+	if spec.Image == nil {
+		return defaultCassandraImageForVersion(spec.Version)
+	}
+	return spec.Image
+
+}
+
+const defaultCassandraImageRepository = "docker.io/cassandra"
```

Comment:
What if this is deployed by people who have a policy to only use images mirrored to an internal repository, but they don't want to manually specify an image tag? Maybe a later patch could make this configurable.

Reply:
I created https://github.com/jetstack/navigator/issues/266